iot-e2e-tests/android/app/src/androidTest/java/com/microsoft/azure/sdk/iot/android/helper/DummyAndroidRunner.java

```diff
@@ -5,28 +5,13 @@
 
 package com.microsoft.azure.sdk.iot.android.helper;
 
-import com.microsoft.appcenter.espresso.Factory;
-import com.microsoft.appcenter.espresso.ReportHelper;
-import com.microsoft.azure.sdk.iot.common.helpers.BasicTierOnlyRule;
-import com.microsoft.azure.sdk.iot.common.helpers.ConditionalIgnoreRule;
-
-import org.junit.After;
-import org.junit.Rule;
 import org.junit.Test;
 
-// This class is to add a dummy test for every TestGroup.  Currently, App Center errors out
-// when a TestGroup did not execute any actual tests.
+// This class is to add a dummy test for every TestGroup.  Currently, android testing errors out
+// when a TestGroup did not execute any actual tests, so this forces each test group to have at least
+// this test
 public class DummyAndroidRunner
 {
```

Comment:
Does `TestSuite` annotation work better with emulators? That way we wouldn't need a fakeTest class and a lot of other overhead code.

Reply:
We do still need this. We added this fake test class for App Center, but they share the same problem as running the test on an emulator. Anytime you invoke instrumentation tests on an android device, emulator or real, at least one test needs to run. Otherwise it will report an error

Reply:
Sure, but with `TestSuite`, instead of multiple runners, we can have a single runner that references all the test files, and does the initial setup.

Reply:
I don't see how we could split the tests into groups with this approach